docs/src/pages/components/cards/cards.md

```diff
@@ -11,6 +11,8 @@ components: Card, CardActionArea, CardActions, CardContent, CardHeader, CardMedi
 
 They should be easy to scan for relevant and actionable information. Elements, like text and images, should be placed on them in a way that clearly indicates hierarchy.
 
+👑 If you are looking for inspiration, you can check [MUI Treasury's customization examples](https://mui-treasury.com/components/card).
```

Comment:
Ah, we have no customization demo for this component. What about we link it at the bottom of the page (`## Complementary projects`)?